package.json

```diff
@@ -70,18 +70,17 @@
         "@inkline/cli": "^1.2.1",
         "@inkline/config": "^1.8.0",
         "@playwright/test": "^1.37.1",
-        "@storybook/addon-a11y": "^7.5.3",
-        "@storybook/addon-actions": "^7.5.3",
-        "@storybook/addon-essentials": "^7.5.3",
-        "@storybook/addon-interactions": "^7.5.3",
-        "@storybook/addon-links": "^7.5.3",
-        "@storybook/addon-postcss": "^2.0.0",
-        "@storybook/addon-styling": "^1.3.7",
-        "@storybook/manager-api": "^7.5.3",
+        "@storybook/addon-a11y": "^7.6.0-alpha.7",
```

Comment:
we're using the prerelease to test that it works. Once `v7.6.0` is released to stable next week I'll update these to match and drop the `overrides` below.
I'll also take the PR out of draft by then.